components/LandingPage/MusingsFromTheTeam/MusingsFromTheTeam.tsx

```diff
@@ -0,0 +1,72 @@
+import styles from './MusingsFromTheTeam.module.scss';
+import type { BlogPost } from 'types/Base';
+import Image from '../../Image';
+
+type MusingsFromTheTeamProps = {
+    postData: BlogPost[];
+};
+
+const BlogPostAuthor = ({ post }: BlogPostCardProps) => (
+    <div className={styles.author}>
+        <Image
```

Comment:
***Suggestion:*** Can we optimize our images, looks like lisa's is 160kb alone and fetching at 1920p? Looks like we should be able to define the width and height in the getLandingPage query. (search for "Image transformations" on this page https://www.contentful.com/developers/docs/references/graphql/#/reference/schema-generation/assets)

**Thought:** Looks like we could make more use of NextJS image component, such as the use of the [priority](https://nextjs.org/docs/api-reference/next/image#priority) prop for hero images.

Reply:
I would like to refactor my image component across the site to make use of the `sizes` option, and specify some good default sizes in the next.js config file ([more information about this in this thread](https://github.com/vercel/next.js/issues/18413)). This might take another ticket to look into across the site